package.json

```diff
@@ -98,14 +98,14 @@
   },
   "scripts": {
     "build": "gatsby build",
-    "build:lambda": "netlify-lambda build src/lambda --config=./webpack.lambda.js",
+    "build:lambda": "NODE_OPTIONS=--openssl-legacy-provider netlify-lambda build src/lambda --config=./webpack.lambda.js",
```

Comment:
`netlify-lambda` uses some hashing function that is failing with node v18. We need to add this flag in order to make it work.